docs/benchmark/ftlib_benchmark.md

```diff
@@ -0,0 +1,171 @@
+## Benchmark of FTlib AllReduce
+
+We perform experiments to test the performance of FTlib AllReduce in
+ElasticDL on three Kubernetes clusters -- a Minikube cluster, an on-premise CPU
+cluster managed by a tailored Kubernetes system, and a GPU cluster
+managed by the same tailored Kubernetes system. In those experiments,
```

Comment:
In those experiments, we adopt ResNet50 and MobileNetV2 models to test the performance of computation and communication. ResNet50 is a computation-intensive model and MobileNetV2 is a communication-intensive model.

=>

We train a computation-intensive model, ResNet50, and a communication-intensive one, MobileNetV2, in the benchmark.